app/views/items/show.html.erb

```diff
@@ -50,18 +50,18 @@
       <img class="avatar" alt="avatar" src="https://kitt.lewagon.com/placeholder/users/cveneziani" />
 
       <div class="card-description">
-        <p class="item-name">From <%= @item.name %></p>
-        <p class="username"> <%= @item.user.username %></p>
+        <p class="item-name"> <%= @item.name.capitalize %></p>
```

Comment:
I think Williams fixed this already but I also added capitalize. (maybe a small merge conflict with this line)